server/middleware/populateCurrentUser.ts

```diff
@@ -7,8 +8,9 @@ export default function populateCurrentUser(userService: UserService): RequestHa
     try {
       if (res.locals.user) {
         const user = res.locals.user && (await userService.getUser(res.locals.user.token))
```

Comment:
I know this was in the template but this line confused me a bit. Can we remove the check for `res.locals.user &&` because it's already checked that in the enclosing if statement?